lib/GLPI/Agent/Task/Inventory/Generic/Hostname.pm

```diff
@@ -21,9 +21,15 @@ sub doInventory {
     my (%params) = @_;
 
     my $inventory = $params{inventory};
-
-    # keep just the hostname
-    my $hostname = getHostname(short => 1);
+    my $assetname_support = $params{'assetname_support'};
+
+    # use the hostname as desired
+    my $hostname;
+    if ($assetname_support == 2 ) {
```

Comment:
There's one space to remove here.
```suggestion
    if ($assetname_support == 2) {
```

Reply:
Concerning windows: since by default it tries to use the DNS name, I think changing that might not be desired. Also, I don't think windows has a notion of fqdn as hostname, so "short" being 0 or 1 will not change anything (and next to that: I don't have direct access to check this as admin on windows servers).
So, for now: I could put in the Changes file that this only impacts unix-flavored OSs.

Reply:
I adapted the Changes file (and others) to indicate the option does not apply on MacOS and Windows.
And fixed the spaces.
Also: in Win32/OS.pm there is also $hostname being defined, but never used ==> should I remove it there?